### PR TITLE
My organisation.UI

### PR DESCRIPTION
**Pull request by Raisa-12**
### Functionality:
Fix the popup UI of edit organization profile and documents section in dark theme.

### Solution:
made changes in edit-dp.component.css,edit-dp.component.html,edit-dp.component.ts,lib.js,,libranian.js,view-organization-details.component.html,view-organization-details.component.ts,auth.service.ts,

### Risk level:
- [X] high 
- [ ] medium
- [ ] low

### How to test:
change the edit profile of My Organisation Page

### CODE REVIEW

File: src/app/body/view-organization-details/edit-dp/edit-dp.component.ts [hunk 1]
```diff
@@ -11,85 +13,89 @@ export class EditDpComponent implements OnInit {
...
+    if(this.imageUrl== "") {
+      this.enableCropper = false
+    } else {
+      this.enableCropper = true
     }
-    reader.readAsDataURL(file)
+    this.imageUrlTemp = this.imageUrl;
+    console.log(this.imageUrlTemp);
 
+    this.basePath = '/Organizations/' + this.orgDomain + '/Logo';
```
Comment (abhishek-mishra019) [inline src/app/body/view-organization-details/edit-dp/edit-dp.component.ts:48]: have to replace logo by LogoFIles

File: src/app/body/view-organization-details/view-organization-details.component.ts [hunk 2]
```diff
@@ -74,6 +79,16 @@ export class ViewOrganizationDetailsComponent implements OnInit {
     });
   }
 
+  readOrganizationLogo(orgDomain: string) {
+    this.authService.getOrganizationLogo(orgDomain).subscribe(fileData => {
+      if(fileData[fileData.length-1] != undefined) {
+        this.imageUrl = fileData[fileData.length-1].FileUrl;
+        this.profilePicFile = fileData[fileData.length-1];
```
Comment (abhishek-mishra019) [inline src/app/body/view-organization-details/view-organization-details.component.ts:86]: replace name of profilePicFile by orgLogoFile

**Comment by abhishek-mishra019** [comment]
NOT COMPLETED, NEED TO COMPLETE THIS BEFORE RELEASE

**Review by abhishek-mishra019 — COMMENTED** [review]
Not completed, will complete the changes before the next release

**Comment by viv9k** [comment]
This pr can be closed

**Pull request closed without merging**